.azure-pipelines-templates/matrix.yml

```diff
@@ -1,188 +1,83 @@
 parameters:
-  consensus: ['CFT', 'BFT']
   target: ['NoSGX', 'SGX']
 
   env:
-    Checks:
+    Hosted:
       container: nosgx
-      pool: Ubuntu-1804-D16s_v3
+      pool:
+        vmImage: ubuntu-18.04
     NoSGX:
       container: nosgx
       pool: Ubuntu-1804-D16s_v3
     SGX:
       container: sgx
       pool: Ubuntu-1804-DC8_v2
-    SAN:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-    Release:
-      container: sgx
-      pool: Ubuntu-1804-DC8_v2
-    Metrics:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-      dependsOn: ['Test_SGX_CFT_Perf', 'Test_SGX_BFT_Perf']
-    Publish:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-      dependsOn: ['Test_SGX_CFT_Release', 'Test_SGX_BFT_Release']
     Perf:
       pool: CCF-Perf
 
   build:
     common:
       cmake_args: '-DCMAKE_C_COMPILER_LAUNCHER="ccache" -DCMAKE_CXX_COMPILER_LAUNCHER="ccache"'
     NoSGX:
-      cmake_args: '-DCOMPILE_TARGETS=virtual -DCOVERAGE=ON'
+      cmake_args: '-DTARGET=virtual'
     SGX:
       cmake_args: ''
     debug:
-      cmake_args: '-DCMAKE_BUILD_TYPE=Debug -DVERBOSE_LOGGING=ON -DBUILD_SMALLBANK=OFF'
+      cmake_args: '-DCMAKE_BUILD_TYPE=Debug -DBUILD_SMALLBANK=OFF'
     install:
       install_prefix: '/tmp/ccf-install'
       cmake_args: '-DCMAKE_INSTALL_PREFIX=/tmp/ccf-install'
-    san:
-      cmake_args: '-DSAN=ON'
     perf:
       cmake_args: '-DDISTRIBUTE_PERF_TESTS="`../.nodes.sh`"'
 
   test:
     NoSGX:
-      BFT:
-        ctest_args: '-LE "benchmark|perf|suite|raft"'
-      CFT:
-        ctest_args: '-LE "benchmark|perf|suite|pbft"'
+      ctest_args: '-LE "benchmark|perf|suite" -j2'
     SGX:
-      BFT:
-        ctest_args: '-LE "benchmark|perf|raft"'
-      CFT:
-        ctest_args: '-LE "benchmark|perf|pbft"'
+      ctest_args: '-LE "benchmark|perf" -j2'
     Release:
-      BFT:
-        ctest_args: '-LE "benchmark|perf|raft"'
-      CFT:
-        ctest_args: '-LE "benchmark|perf|pbft"'
+      ctest_args: '-LE "benchmark|perf" -j2'
```

Comment:
If we're doing a release, should we just run the full test suite?

Reply:
Sure, but unless it's also Perf, we don't record the values. Seems fair though.